exercise/views.py

```diff
@@ -143,6 +149,10 @@ def post(self, request, *args, **kwargs):
                     LearningObject.STATUS.ENROLLMENT_EXTERNAL,
                 ) and new_submission.status == Submission.STATUS.READY:
                     self.instance.enroll_student(self.request.user)
+                    # Save a link to the latest submission of an enrollment
+                    # exercise per user.
+                    self.profile.latest_enrollment_submission = new_submission
```

Comment:
Can you make method for this. E.g. `self.profile.set_latest_enrollment_submission(new_submission) . That way, it's easier to merge user models and to create a proxy for `.profile`.